policyengine_us/parameters/gov/states/ct/tax/income/additions/additions.yaml

```
@@ -0,0 +1,19 @@
+description: Connecticut counts these sources as additions.
+values:
+  2022-01-01:
+    - us_govt_interest
```

i dont think we can include this here, it would not make sense to have it as an addition and as a subtraction 
```suggestion
    - us_govt_interest
```